src/Apps/Order/Components/OfferInput.tsx

```diff
@@ -22,7 +22,7 @@ export const OfferInput: FC<OfferInputProps> = ({
 }) => {
   const formatValueForDisplay = (val: number | undefined) => {
     if (val !== undefined && val > 0) {
-      return val.toLocaleString()
+      return val.toString().replace(/\B(?=(\d{3})+(?!\d))/g, ",")
```

Comment:
I wonder if setting a fixed locale would just work. For example, `val.toLocaleString("en-US")`

Reply:
I was hesitant to put specific "en-US" here but made a search based on your comment and looks like we use hardcoded "en-US" formatting already in other places around Pricing. So this works as well!

Thanks for the nudge.